src/highlevelcrypto.py

```diff
@@ -9,10 +9,16 @@
 
 from binascii import hexlify
 
-import pyelliptic
-from bmconfigparser import BMConfigParser
-from pyelliptic import OpenSSL
-from pyelliptic import arithmetic as a
+try:
```

Comment:
I'm not sure about this. Need to think about it.

Reply:
You don't need this, please look into #1788

Reply:
However in tests the imports gonna be like these. See: a8c79c3832e2e3081ebdcb0373f6b4c0057906f7.

Reply:
Okay, got it